src/scripts.js

```diff
@@ -1,15 +1,20 @@
 import './styles.css';
-import apiCalls from './apiCalls';
-import Ingredient from './classes/Ingredient';
-import Recipe from './classes/Recipe';
 import RecipeRepository from './classes/RecipeRepository';
 import User from './classes/User';
-import recipeData from './data/recipes';
-import ingredientsData from './data/ingredients';
-import usersData from './data/users';
+import {usersApi, recipesApi, ingredientsApi} from './apiCalls';
+
+let user;
+let recipeRepo;
+
+Promise.all([usersApi, recipesApi, ingredientsApi])
```

Comment:
The promise.all will resolve only once all three of the API calls resolve. The downside to using this method would be if one call resolves to an error, none of the calls will work, since we've attached the .catch to all of them at once.